damage_calculator.py

```diff
@@ -313,14 +327,14 @@ def calc_dps(client, god, build, enemy, enemy_build, enemy_level, level=20):
                 elif "Spectral Armor" in enemy_build and "Deathbringer" in build:
                     dmg += attDamage * (1.3 - .55)
                 else:
-                    dmg += attDamage
+                    dmg = attDamage * 1.75
                 if "Wind Demon" in build and not flag:
                     pen_per += 10
                     flag = True
 
         if (i > 0 and i % 4 == 0) and "Odysseus' Bow" in build:
-            item_dmg += round(15 + (power * .6))
-            item_dmg_out["Odysseus' Bow"] += round(15 + (power * .6))
+            item_dmg += round(15 + (calc_auto_dmg(god, attDamage) * .6)) 
```

Comment:
verify this works correctly in game

Reply:
verified ody bow is based off atk damage not power if thats what you mean

Reply:
youd have to check in jungle practice to see which way is correct in terms of damage numbers

Reply:
item desc says basic attack damage which is good enough imo

Reply:
checked in game, applies BA dmg